src/main/java/org/jenkinsci/plugins/azurekeyvaultplugin/AzureKeyVaultGlobalConfiguration.java

```diff
@@ -100,15 +100,19 @@ private Optional<String> resolveCredentialIdFromEnvironment() {
         String uami = getPropertyByEnvOrSystemProperty("AZURE_KEYVAULT_UAMI_ENABLED", "jenkins.azure-keyvault.uami.enabled")
                 .orElse("false");
 
+        CredentialsScope scope = getPropertyByEnvOrSystemProperty("AZURE_KEYVAULT_SP_SCOPE", "jenkins.azure-keyvault.sp.scope")
```

Comment:
should use the same uami namespace I think:
```suggestion
        var isUami = uami.equals("true");
        CredentialsScope scope = getPropertyByEnvOrSystemProperty("AZURE_KEYVAULT_SP_SCOPE", isUami ? "jenkins.azure-keyvault.uami.scope" : "jenkins.azure-keyvault.sp.scope")
```